### ISSUE TITLE
One more character selected in visual mode

### ISSUE THREAD

**Issue by AlexGuo1998** [jaredpar/VsVim#2360]
When using visual mode (key v), one more character is selected under the cursor.

This behavior can be annoying, since vim only select the character under the cursor when no other character(s) are selected.

Example: `abcdefg` in the buffer, and cursor pointing at `d`.

Action|vim behavior|VsVim behavior
-|-|-
`vy`|`d` selected and copied|same as vim
`vly`|`d` selected and copied|`de` selected and copied
`vhy`|`c` selected and copied|`cd` selected and copied

**Comment by ricksladkey** [comment]
Not sure what vim settings you are talking about or vim version, but my testing shows that gvim 8.0 does not behave as you describe for either `:set selection=exclusive` or `:set selection=inclusive`. 

| Selection Setting | Action | vim behavior | VsVim behavior |
|---|---|---|---|
| `exclusive` | `vy` |  nothing selected, `d` copied | nothing selected, empty string copied |
| `exclusive` | `vly` |  `d` selected and copied | same |
| `exclusive` | `vhy` |  `c` selected and copied | same |
| `inclusive` | `vy` |  `d` selected and copied | same |
| `inclusive` | `vly` |  `de` selected and copied | same |
| `inclusive` | `vhy` |  `cd` selected and copied | same |

In short, vim always copies at least one character, even when nothing is selected, and in all other situations vim and VsVim behave the same. So there is one slight difference from vim and yet the VsVim behavior is arguably more intuitive, albeit incompatible with vim.

**Comment by AlexGuo1998** [comment]
Oh, I see...

It turned out to be the command in my vimrc `behave mswin` changed the selection behavior. At first, I thought that VsVim won't support that command, and I just comment out that line in my vsvimrc. So I'm using `exclusive` in vim and `inclusive` in VsVim. That's the difference.

After all, thanks for explaining!
